### ISSUE TITLE
Improve error log when WP update fails

### ISSUE THREAD

**Issue by jonathanstegall** [MinnPost/object-sync-for-salesforce#413]
**Is your feature request related to a problem? Please describe.**
When a pull action is unable to update a WordPress record (for example, because it no longer exists) the log message title reads like this:

> Error: WordPress update for user ID [ID number] was unsuccessful with these errors:

And then the log message reads like this:

```php
WP_Error Object
(
[errors] => Array
(
[invalid_user_id] => Array
(
[0] => Invalid user ID.
)

)

[error_data] => Array
(
)

[additional_data:protected] => Array
(
)

)
```

The problem is that it's not clear what record in Salesforce caused the error to happen.

**Describe the solution you'd like**

I think either in the log title, in the log message, or both, we should show what Salesforce record was trying to do the pull operation. This way admins could go look in Salesforce for the record and see what to do with it.

I notice that this case also doesn't create a Mapping Error in the plugin admin. Maybe it should.

**Comment by jonathanstegall** [comment]
I think we could add all of the `$params` object to the message body and see what it contains. I can't remember if it has the Salesforce record ID, but at least it would have whatever info about the record the plugin is trying to save.

**Comment by jonathanstegall** [comment]
Closed in #417.
